### PR TITLE
Update Hungarian language

### PR DESCRIPTION
**Pull request by AntoszHUN**
Hello!

I don't know who started updating the Hungarian language, but I now dared to take over the correction and updating of the translation.

In my experience, there were many gaps and several incorrect translations in the file.

  - I corrected incorrect translations. (However, there may still be errors.)
  - I have included translations of the newly added new options.
  - I have translated the old and new error messages that appear.
  - The translation is currently 100% ready.

Signed-off-by: AntoszHUN <zsikaiantal@gmail.com>

### CODE REVIEW

**Comment by pedrolamas** [comment]
Thank you for this, @AntoszHUN!

If I may ask, can you please change the footer in your original comment above to `Signed-off-by: Your Name <your email address>` following the [contribution rules](https://github.com/fluidd-core/fluidd/blob/develop/CONTRIBUTING.md)?

**Comment by AntoszHUN** [comment]
> Thank you for this, @AntoszHUN!
> 
> If I may ask, can you please change the footer in your original comment above to `Signed-off-by: Your Name <your email address>` following the [contribution rules](https://github.com/fluidd-core/fluidd/blob/develop/CONTRIBUTING.md)?

Thanks! I corrected. I apologize for the mistake afterwards!

**Review by matmen — APPROVED** [review]

File: src/locales/hu.yaml [hunk 1]
```diff
@@ -67,59 +78,97 @@ app:
...
     tooltip:
-      low_on_space: Kevés lemezterület
+      low_on_space: Kevés a lemezterület
+      root_disabled: '{root} root nem érhető el. Kérjük, ellenőrizze a naplóit.'
+  gcode:
+    btn:
+      load_current_file: Az aktuális fájl betöltése
+    label:
+      current_layer_height: Jelenlegi rétegmagasság
+      follow_progress: Kövesse az előrehaladást
```
Comment (matmen) [inline src/locales/hu.yaml:108]:
```suggestion
      follow_progress: Folyamat nyomon követése
```

File: src/locales/hu.yaml [hunk 2]
```diff
@@ -132,184 +181,319 @@ app:
...
       extrude_speed: Extrudálás sebessége
+      filament: Nyomtatószál
+      file: Fájl
+      finish_time: Befejezés
       flow: Átfolyás
       free: szabad
+      heaters_busy: A nyomtató jelenleg foglalt. A fűtőtestek kikapcsolása sikertelen nyomtatást eredményezhet.
       high: Magas
-      host: Gazda
+      host: Házigazda
```
Comment (matmen) [inline src/locales/hu.yaml:213]:
```suggestion
      host: Gazdagép
```

File: src/locales/hu.yaml [hunk 3]
```diff
@@ -1,37 +1,48 @@
 app:
   bedmesh:
     label:
-      profile_name: Profil neve
-      remove_profile: '%{name} profil eltávolítása'
+      box_scale: Dobozmérleg
```
Comment (matmen) [inline src/locales/hu.yaml:4]:
```suggestion
      box_scale: Skála
```

File: src/locales/hu.yaml [hunk 4]
```diff
@@ -132,184 +181,319 @@ app:
...
+        apply_z_offset_endstop: >-
+          A végpont Z-eltolása az aktuális értékből lesz számítva és frissítve
+          a Z-eltolás.
+        apply_z_offset_probe: >-
+          A Probe/Bltouch Z-eltolás kiszámítása és frissítése az aktuális értékből
+          történik.
     table:
       header:
-        actions: Lehetőségek
+        actions: Akciók
```
Comment (matmen) [inline src/locales/hu.yaml:285]:
```suggestion
        actions: Parancsok
```

File: src/locales/hu.yaml [hunk 5]
```diff
@@ -132,184 +181,319 @@ app:
...
+      unretract_speed: Visszahúzási sebesség
       used: használt
-      variance: Variálás
+      variance: Eltérés
       velocity: Sebesség
-      z_offset: Z eltolás
+      visible: Látható
+      z_offset: Z Eltolás
+      unsaved_changes: Nem mentett módosítások
+      alias: Álnév
```
Comment (matmen) [inline src/locales/hu.yaml:253]:
```suggestion
      alias: Becenév
```

**Review by matmen — CHANGES_REQUESTED** [review]
I've had a Hungarian friend double check these and they suggested some changes. Please let me know what you think.

File: src/locales/hu.yaml (hunk 5 above)
Comment (AntoszHUN) [reply src/locales/hu.yaml:253]: Well, I remember writing that there were still bugs in it. I am constantly working to improve them.

Thank you for your understanding!

By: AntoszHUN

**Comment by AntoszHUN** [comment]
I made another fix. Update successful. I corrected the incorrect translations!

By: AntoszHUN

**Review by pedrolamas — APPROVED** [review]

**Review by matmen — APPROVED** [review]

**Pull request merged**